change/@ni-jasmine-parameterized-b84dc27d-fa72-4663-9049-3d8ddf8c7deb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I find these two commits concerning:
![image](https://github.com/user-attachments/assets/af36fc66-41dc-4383-99b1-14566f27317e)

It sounds like the repo is in a bad state where upgrading to the latest dependencies that our package.json says we are compatible with breaks the build. Looks like something was done to manipulate the lockfile to get the PR to pass.

Instead we should but effort to get the repo in a good state (such as pin dependencies and file issues about breakages with the latest versions). I'll switch gears and do as it seems important to have the dependencies and lock file be stable before a large dependency update.

Reply:
I'm in the process of trying to get a minimal set of dependencies upgraded. I want to keep this submission as unadulterated as possible.

Reply:
> I want to keep this submission as unadulterated as possible.

Not quite the right solution depending on what you mean. Captured the expected actions in this situation in more detail here: https://github.com/ni/nimble/issues/1479#issuecomment-2237523576